easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb

```diff
@@ -0,0 +1,27 @@
+name = 'psmpi'
+version = '5.1.5-1'
+
+homepage = 'https://github.com/ParaStation/psmpi2'
+description = """ParaStation MPI is an open source high-performance MPI 3.0 implementation,
+based on MPICH v3. It provides extra low level communication libraries and integration with 
+various batch systems for tighter process control.
+"""
+
+toolchain = {'version': '4.9.3', 'name': 'GCC'}
+
+toolchainopts = {'defaultopt': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/ParaStation/psmpi2/archive/']
+
+dependencies = [
+    ('pscom', '5.0.48-1', '', True),
+]
+
+hiddendependencies = [
+    ('pscom', '5.0.48-1', '', True),
```

Comment:
It's just that we currently don't ship any easyconfigs that include `hiddendependencies`.

Granted, we don't have a good way of dealing with site-specific customisations yet...

Maybe we should put some work into the `.ebp` idea, cfr. https://github.com/hpcugent/easybuild-framework/issues/544